recipes/fiji-ilastik/post-link.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Version 1.4.0b29
+ilastik_bin=${CONDA_PREFIX}/bin/ilastik
+if [ ! -e "$ilastik_bin" ]; then
+  # Version 1.4.0b27
+  ilastik_bin=${CONDA_PREFIX}/bin/ilastik-app
+  if [ ! -e "$ilastik_bin" ]; then
+    ilastik_bin=""
+  fi
+fi
```

Comment:
so sorry that you had to go through this :_( . I think for simplicity we could maybe skip versions <1.4.0b29 ? So that we wouldn't need to carry around this legacy stuff...

Reply:
No problem. Updated